### ISSUE TITLE
RGAA 3.3.1 -  Color contrast falling for "the color should be between (...)"

### ISSUE THREAD

**Issue by dzc34** [Asqatasun/Contrast-Finder#10]
## Observed behaviour

![selection_002](https://cloud.githubusercontent.com/assets/6709977/21913420/9b4f07e8-d92c-11e6-982c-2f2ec9ad7c14.png)

## Expected behaviour
The contrast ratio between text and its background must be at least 4.5:1

[New colors thanks to Contrast-finder](https://app.contrast-finder.org/result.html?foreground=%23737373&background=%23F6F6F6&isBackgroundTested=false&ratio=4.5&algo=HSV)  itself
